docs/src/main/sphinx/release/release-390.md

```diff
@@ -0,0 +1,52 @@
+# Release 390 (13 July 2022)
+
+## General
+
+* Add support for [setting comments on views](/sql/comment). ({issue}`8349`) ({issue}`8349`)
+* Improve performance of queries with an `UNNEST` clause. ({issue}`10506`)
+* Fix potential query failure when forced spilling joined is enabled by the
+  `force-spilling-join-operator` configuration property or the
+  `force_spilling_join` session property. ({issue}`13123`)
+* Fix incorrect results for certain join queries containing filters involving
+  explicit or implicit casts. ({issue}`13145 `)
+
+## Cassandra connector
+
+* Change mapping for Cassandra `inet` type to Trino `ipaddress` type.
+  Previously, `inet` was mapped to `varchar`. ({issue}`851`)
+* Remove the unused configuration properties
```

Comment:
If they were unused, why do we need to add an entry?

Reply:
Does it not cause an error if someone has a particular configuration property set and the property no longer exists?

Reply:
It depends on how the config options were being processed. Given that connectors generally user Airlift property validations, yes, users will get an error if they were previously using those options.

The wording sounds strange, though. It makes it sound as if there removal won't have any effect, since... well... the properties are unused.

We may want to rephrase as "Remove support for the xxx configuration properties", or something to that effect.